capa/main.py

```diff
@@ -256,6 +256,11 @@ def find_capabilities(ruleset: RuleSet, extractor: FeatureExtractor, disable_pro
     with redirecting_print_to_tqdm(disable_progress):
         with tqdm.contrib.logging.logging_redirect_tqdm():
             pbar = tqdm.tqdm
+            if capa.helpers.is_runtime_ghidra():
```

Comment:
The call to `is_runtime_ghidra` is causing our tests to fail. I believe this is because we search for `ghidra.program.flatapi` when we should be searching for `ghidra`. What do you think @colton-gabertan ?

Reply:
The change worked :). Thank you for catching that.